speaker/bgp_controller_test.go

```diff
@@ -211,7 +213,7 @@ func TestBGPSpeaker(t *testing.T) {
 	}
 	newBGP = b.NewSessionManager
 	c, err := newController(controllerConfig{
-		MyNode:        "pandora",
+		MyNode:        nodeName,
```

Comment:
can we restore the previous version? I think having the explicit name is more clear